### PR TITLE
[CBRD-20760] introduce macros for cubrid.conf file

### PR DESCRIPTION
**Pull request by funkydvd**
http://jira.cubrid.org/browse/CBRD-20760
I have introduced macros for min, max and default, and also made a new field, id, in the sysprm_param structure, for accessing easier the prm_Def[] structure.

### CODE REVIEW

File: src/base/system_parameter.c [hunk 1]
```diff
@@ -8089,6 +8382,35 @@ sysprm_generate_new_value (SYSPRM_PARAM * prm, const char *value, bool check, bo
     }
 #endif /* SERVER_MODE */
 
+  if (strcasecmp (value, PRM_VALUE_DEFAULT) == 0)
+    {
+      set_default = true;
```
Comment (eseokoh) [inline src/base/system_parameter.c]: Is it possible to change `set_default` as an auto variable rather than a parameter of function?

Comment (funkydvd) [inline src/base/system_parameter.c]: Yes. It looks like the parameter was used only in one place, and it checked the same thing (the value to be "DEFAULT")

File: src/base/system_parameter.c [hunk 2]
```diff
@@ -7726,29 +7926,10 @@ xsysprm_dump_server_parameters (FILE * outfp)
...
 
-/*
- * sysprm_get_range -
- *   return:
- *   pname (in): parameter name
- *   value (in): parameter value
- */
 int
-sysprm_get_range (const char *pname, void *min, void *max)
+sysprm_get_param_range (SYSPRM_PARAM * prm, void *min, void *max)
```
Comment (ribram) [inline src/base/system_parameter.c]: Please add function description.

File: src/base/system_parameter.c [hunk 3]
```diff
@@ -7847,6 +8028,31 @@ sysprm_get_range (const char *pname, void *min, void *max)
 }
 
 /*
+ * sysprm_get_range -
+ *   return:
+ *   pname (in): parameter name
+ *   value (in): parameter value
```
Comment (ribram) [inline src/base/system_parameter.c]: Fix description for sysprm_get_range

**Pull request merged**